src/test/java/com/google/javascript/gents/multiTests/module_imports/export_default.ts

```diff
@@ -1,3 +1,3 @@
-export default class A {
```

Comment:
This is quite useful - it'll make sure that JavaScript VMs assign a name to the function, visible in stack traces etc. Same for the case above.

Is this causing a problem?